db-server/test/backend/db_tests.js

```diff
@@ -1956,6 +1956,30 @@ module.exports = function (config, DB) {
           })
       })
 
+      it('should update session verificationMethod', () => {
+        const verifyOptions = {
+          verificationMethod: 'totp-2fa'
+        }
+        return db.verifyTokens(sessionToken.tokenVerificationId, account)
+          .then(() => {
+            return db.sessionToken(tokenId)
+          }, assert.fail)
+          .then((token) => {
+            assert.equal(!! token.mustVerify, false, 'mustVerify is null')
```

Comment:
The comment here suggest we expect it to be `null`, but then for the test we coerce it to a boolean and compare it to `false`, which seems kinda funny.

If the comment is true why aren't we comparing to `null`? Or if the comment is wrong and we expect a `boolean`, why are we coercing it first? Is it expected that it might be `null` or `false`? Or can we guarantee that it will always be a boolean and eliminate the coercion? The stricter the tests are the better, is all I'm thinking.

Reply:
Whoops you are right, this doesn't need coercing. It just needs to check that the value is false.